### PR TITLE
[TUBEMQ-81] Fix LogClearRunner bug && format BrokerDefMetadata

### PR DESCRIPTION
**Pull request by Technoboy-**

### CODE REVIEW

File: tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java [hunk 1]
```diff
@@ -703,7 +703,7 @@ public void run() {
                         .append(tubeConfig.getLogClearupDurationMs()).toString());
                 sBuilder.delete(0, sBuilder.length());
             }
-            if (!expiredTopic.isEmpty()) {
+            if (expiredTopic.isEmpty()) {
```
Comment (aloyszhang) [inline tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java:706]: Print log here indicates that some expired file has been deleted, so I think we cann't make this change .

Comment (aloyszhang) [reply tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java:706]: Maybe move this log print to  [the end of the loop of file delete](https://github.com/apache/incubator-tubemq/blob/master/tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java#L697)

Comment (aloyszhang) [reply tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java:706]: Mabye move this log print to  [the end of file delete loop](https://github.com/apache/incubator-tubemq/blob/master/tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java#L697) is better.

Comment (gosonzhang) [reply tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java:706]: I agree with @aloyszhang ‘s view. If this place is judged to be empty, not only will the previous end-of-print mark for deleting the file log printing be lost, but also the useless printing log will increase(most of the time, the expired file check returns empty).

Comment (Technoboy-) [reply tubemq-server/src/main/java/org/apache/tubemq/server/broker/msgstore/MessageStoreManager.java:706]: Yes, you are right.  I made a mistake of the end of the If logic. I will repost a pull request

**Pull request closed without merging**